src/Microsoft.Azure.SignalRService/HubLifetimeManagerBase.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
+
+namespace Microsoft.Azure.SignalRService
+{
+    internal abstract class HubLifetimeManagerBase : HubLifetimeManager<Hub>, IHubLifetimeManagerForUserGroup
```

Comment:
Better name

Reply:
Delete the class, since it is not so useful